### PR TITLE
MP-2175 Added /suggest-address RPC endpoint

### PR DESCRIPTION
**Pull request by NickVries**
**Related issues**
- https://myparcelcombv.atlassian.net/browse/MP-2175

### CODE REVIEW

File: specification/paths/SuggestAddress.json [hunk 1]
```diff
@@ -0,0 +1,104 @@
+{
+  "post": {
+    "tags": [
+      "RPC"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "addresses:suggest"
```
Comment (Shodske) [inline specification/paths/SuggestAddress.json]: Should be `addresses.suggest`

File: specification/paths/SuggestAddress.json [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                  "country_code": {
+                    "type": "string",
+                    "example": "NL"
+                  },
+                  "postal_code": {
+                    "type": "string",
+                    "example": "2131 BC"
+                  },
+                  "street_number": {
+                    "type": "string",
```
Comment (Shodske) [inline specification/paths/SuggestAddress.json]: Should be a number and I think we should add `street_number_suffix` as a string.

Comment (NickVries) [reply specification/paths/SuggestAddress.json]: After discussing this with the team, we decided to leave the `street_number` property as a string (some street numbers include letters: I live on street number 3A with suffix 1).
In order to be consistent we should probably change types of the street number properties in other resources to string as well, but that will break things and seems like a separate issue.

I did add the `street_number_suffix` property.

Comment (Shodske) [reply specification/paths/SuggestAddress.json]: I think that keeping it a string will cause more problems than that are solved by it. In your case I would assume that you live on number 3 with suffix A-1 (or A1, A/1, etc). If we allow any string to be entered we would always have to check that no suffixes are present in the number.

Another thing is that the PostNL API also requires the street number to be a number. Which means that if we were to take your case with `"street_numer": "3A"` and `"street_number_suffix": "1"`, we would still have to strip the `A` from the number and add it to the `1` from the suffix. So separating these two fields would have no benefit for us if we don't put constraints on them.

**Pull request merged**